addons/sale_project/static/tests/tours/project_create_sol_tour.js

```diff
@@ -0,0 +1,45 @@
+import { registry } from "@web/core/registry";
+import { stepUtils } from "@web_tour/tour_service/tour_utils";
+
+registry.category("web_tour.tours").add('project_create_sol_tour', {
+    test: true,
+    url: "/web",
+    steps: () => [stepUtils.showAppsMenuItem(),
+{
```

Comment:
This part of the code should be indented, like [here](https://github.com/odoo/odoo/blob/17.0/addons/project/static/tests/tours/project_tour.js)